eo-maven-plugin/src/test/java/org/eolang/maven/LintMojoIT.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import com.jcabi.manifests.Manifests;
+import com.yegor256.MayBeSlow;
+import com.yegor256.Mktmp;
+import com.yegor256.MktmpResolver;
+import com.yegor256.WeAreOnline;
+import com.yegor256.farea.Execution;
+import com.yegor256.farea.Farea;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * Integration tests for {@link LintMojo}.
+ *
+ * @since 0.52
+ */
+@SuppressWarnings({"JTCOP.RuleAllTestsHaveProductionClass", "JTCOP.RuleNotContainsTestWord"})
+@ExtendWith({WeAreOnline.class, MktmpResolver.class, MayBeSlow.class, RandomProgramResolver.class})
+final class LintMojoIT {
+
+    /**
+     * The path to the EO file.
+     */
+    private static final String EO = "src/main/eo/foo.eo";
```

Comment:
@Suban05 let's inline this variable since it's used less than 5 times in the code